CHANGES.md

```diff
@@ -4,16 +4,17 @@ This file is only intended to serve the developers of this repository and of the
 
 This provides an overview of the releases that have been made using this release pipeline.
 
-## Work In Progress - April: 2023: Security releases for 1.38.6
+## Work In Progress - July 2023: Security releases for 1.38.7
 
-- [MediaWiki security release announcement](https://lists.wikimedia.org/hyperkitty/list/wikitech-l@lists.wikimedia.org/message/6UQBHI5FWLATD7QO7DI4YS54U7XSSLAN/)
+- [MediaWiki security and maintenance release announcement: 1.35.10 / 1.38.6 / 1.39.3](https://lists.wikimedia.org/hyperkitty/list/wikitech-l@lists.wikimedia.org/message/6UQBHI5FWLATD7QO7DI4YS54U7XSSLAN/)
+- [MediaWiki security and maintenance release announcement: 1.35.11 / 1.38.7 / 1.39.4](https://lists.wikimedia.org/hyperkitty/list/wikitech-l@lists.wikimedia.org/thread/HVT3U3XYY35PSCIQPHMY4VQNF3Q6MHUO/)
 - [MediaWiki full release notes](https://www.mediawiki.org/wiki/Release_notes/1.38)
 
 | Suite Version | MediaWiki release | Date available | Run number | Release task  |
 |---------------|----------------------|----------------|------------| --------------|
-| wmde.12 ([spec](https://github.com/wmde/wikibase-release-pipeline/blob/wmde.12/versions/wmde12.env),[metadata](https://github.com/wmde/wikibase-release-pipeline/blob/main/versions/wmde12)) | MediaWiki (1.38.6, PHP 7.3.19+), WDQS (0.3.118) | April 2023 | ??? | [T334778](https://phabricator.wikimedia.org/T334778) |
+| wmde.12 ([spec](https://github.com/wmde/wikibase-release-pipeline/blob/wmde.12/versions/wmde12.env),[metadata](https://github.com/wmde/wikibase-release-pipeline/blob/main/versions/wmde12)) | MediaWiki (1.38.6, PHP 7.3.19+), WDQS (0.3.118) | July 2023         | ??? | [T334778](https://phabricator.wikimedia.org/T334778) |
```

Comment:
as per https://github.com/wikimedia/mediawiki-docker/blob/5dfcc29a3820d6129124f8eb154a6632d61a3162/1.38/apache/Dockerfile and https://hub.docker.com/layers/library/php/8.0-apache/images/sha256-9de364108aae7369571ce5fd422fc4fbebde89e0a8307b95460db8d4ce16132d?context=explore it should be PHP 8.0.29

Reply:
and it would be cool to add the apache version, in this case: debian/apache2 2.4.56-1~deb11u2 as per https://hub.docker.com/layers/library/php/8.0-apache/images/sha256-9de364108aae7369571ce5fd422fc4fbebde89e0a8307b95460db8d4ce16132d?context=explore